services/nginz/integration-test/conf/nginz/nginx.conf

```diff
@@ -291,7 +291,17 @@ http {
     location /oauth/token {
       include common_response_no_zauth.conf;
       proxy_pass http://brig;
-    }    
+    }
+
+    location /oauth/revoke {
+      include common_response_no_zauth.conf;
+      proxy_pass http://brig;
+    }  
+
+    location /oauth/applications {
+      include common_response_with_zauth.conf;
+      proxy_pass http://brig;
+    }            
```

Comment:
shouldn't this be in https://github.com/wireapp/wire-server/pull/2989?  why not?  (same for [charts/nginz/values.yaml](https://github.com/wireapp/wire-server/pull/3101/files#diff-40768c765dccb009ab75954e5cdcaf502510b04989fe3e4220eb77c748aa7b36))

Reply:
yes, it is